src/components/common/Tooltip/Tooltip.tsx

```diff
@@ -1,29 +1,34 @@
+import { useState } from 'react';
 import styles from './tooltip.module.scss';
 
 type TooltipProps = {
-    isVisible: boolean;
-    textToShow: string;
-    tooltipPosition: {
-        top?: string;
-        right?: string;
-        bottom?: string;
-        left?: string;
-    };
+    children: React.ReactNode;
+    content: React.ReactNode;
+    tooltipPosition?: React.CSSProperties;
```

Comment:
can we have it like this?

```suggestion
    children: React.ReactNode | string;
    tooltipPosition?: React.CSSProperties;
```

Reply:
ReactNode mean both React element as well as plain strings